doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -53,6 +53,36 @@ Serial LTE Modem (SLM)
     * ``AT#XSOCKETOPT=1,53,`` with ``AT#XSOCKETOPT=1,61,4`` to indicate ``RAI_ONGOING``.
     * ``AT#XSOCKETOPT=1,54,`` with ``AT#XSOCKETOPT=1,61,5`` to indicate ``RAI_WAIT_MORE``.
 
+SUIT DFU for nRF54H20
+---------------------
+
+.. toggle::
+
+  * The manifest sequence number is no longer configured through sysbuild Kconfig option.
+    The values are now read from the ``VERSION`` file.
+    Note that this change requires updates in non-default, customized SUIT templates:
```

Comment:
I don't fully understand this sentence: do we really need to specify "non-default, customized"?
```suggestion
    This change to :file:`VERSION` requires the following updates in the SUIT templates for your project:
```

Reply:
We need something else here. Another sentence proposed.